src/server/c-interop-native-extensions.md

```diff
@@ -6,10 +6,15 @@ description: The original way for command-line Dart apps to call C/C++ functions
 _Written by William Hesse <br>
 May 2012_
 
-The extension mechanism discussed in this page is for
-deep integration of the VM.
+{{site.alert.note}}
+The extension mechanism discussed in this page is for deep integration of the VM
+by custom Dart embedders. This extension mechanism is **deprecated**, and may be
```

Comment:
I still really don't like the world `**deprecated**` I would say `**deprecated** for non embedding use cases`

Reply:
Do you like **unsupported** better?

Reply:
I like `**unsupported** for extension purposes` best.